frontend/src/components/MapView/MapView.tsx

```diff
@@ -23,6 +24,7 @@ const MapView = () => {
         const bounds = map.getBounds();
         const centerPoint = map.getCenter();
         let kmDistance: number;
+        if(!filters.isPublic) filters.isPublic = DEFAULT_IS_PUBLIC;
```

Comment:
what is this part for?

Reply:
There are 3 radio buttons here in this filter. The first one is default. So if nothing changed, this default radio must still be sent

Reply:
got it